pkg/elasticsearch/client.go

```diff
@@ -34,6 +34,8 @@ const (
 )
 
 type Client interface {
+	GetClusterName() string
```

Comment:
I recognize this builds upon the established pattern but does not golang conventions typically lead us to just "ClusterName()" and maybe we should consider refactoring the others